WordPressAuthenticator.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressAuthenticator"
-  s.version       = "1.23.0-beta.2"
+  s.version       = "1.23.0-beta.4"
```

Comment:
You'll need to update this since beta.4 is already out.

Reply:
Thank you!